docs/user/openstack/install_upi.md

```diff
@@ -156,6 +161,23 @@ data["compute"][0]["replicas"] = 0;
 open(path, "w").write(yaml.dump(data, default_flow_style=False))'
 ```
 
+### Modify NetworkType (required for Kuryr SDN)
+
+By default the `networkType` is set to `OpenShiftSDN` on the `install-config.yaml`.
+
+If an installation with Kuryr is desired, you must modify the `networkType` field.
+
+This command will do it for you:
+
+```sh
+$ python3 -c '
+import yaml;
```

Comment:
PyYaml might not be installed, it's not in standard library. Maybe we can we just use `sed`?

Reply:
I was following the same structure as previous scripts. I guess we can either highlight the need of installation `PyYaml` or modifying with sed in all other scripts?

Reply:
If there are other scripts like that let's follow it then. :)